Trunk.toml

```diff
@@ -28,6 +28,12 @@ dist = "dist"
 # Optionally perform a cargo clean.
 cargo = false
 
+[binary]
```

Comment:
I'm wondering if perhaps we should call this `ext` or `tools` or something like that. Mainly because when I saw this, it immediately made me think of the Cargo binary / bin sections of the `Cargo.toml`. Not a super big deal, but I figured I would at least bring it up and see what you think.

Reply:
Yeah, I'm not really happy with the naming neither. Went from `download` to `binary` to ... maybe next one is `tools`. `binary` makes sense in the context of downloading binaries but in this context it's confusing as it may mean output binaries of executing trunk as well 🤔

Reply:
+1 for `tools`. `ext` might be confusing with plugins.

Reply:
Sounds like 3 votes for `tools`. That seems pretty reasonable to me.